keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -292,13 +292,22 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+It shouldn't impact already running workloads. It's an opt-in feature,
+and users need to set `pod.spec.topologySpreadConstraints.minDomains` field to use this feature. 
+
+When this feature is disabled by the feature flag, the already created Pod's `pod.spec.topologySpreadConstraints.minDomains` field is preserved,
+but, the newly created Pod's `pod.spec.topologySpreadConstraints.minDomains` field is silently dropped.
+
+
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
 
+A spike on metric `schedule_attempts_total{result="error|unschedulable"}` when pods using this feature are added.
```

Comment:
I would also include a spike on `scheduling_algorithm_duration_seconds`, which would suggest that the feature is too slow.

Reply:
👍 
Added a mention of `plugin_execution_duration_seconds` as well.